x/oracle/types/claim.go

```diff
@@ -1,10 +1,10 @@
 package types
 
 // NewClaim returns a new Claim
-func NewClaim(id string, validatorAddress string, content string) Claim {
-	return Claim{
-		Id:               id,
-		ValidatorAddress: validatorAddress,
-		Content:          content,
-	}
-}
+// func NewClaim(id string, validatorAddress string, content string) Claim {
```

Comment:
If we don't use this code anymore, let's remove this file.

Reply:
accepted.